### DISCUSSION TITLE
Помощь проекту

### DISCUSSION THREAD

**Discussion by completely-burned** [completely-burned/glowing-octo-shame-arma2#133] [General]
[![sv1](https://cache.gametracker.com/server_info/rvtbn5rfvgyhhbyjuh.dynv6.net:2302/b_560_95_1.png)](https://www.gametracker.com/server_info/rvtbn5rfvgyhhbyjuh.dynv6.net:2302/)

**Comment by completely-burned** [comment]
Не стесняйтесь высказать свой отзыв, хочется что-нибудь конструктивное что поможет улучшить ситуацию, т.к. я не могу все проверить. Но мне не нужна оценка на подобии количества звёзд хорошо плохо итп, мне и вам это ничего не даст и никак не поможет.

**Comment by completely-burned** [comment]
Если вы запускали эту миссию, но после этого были обнаружены ошибки в файле [.RPT](https://community.bistudio.com/wiki/Crash_Files), то желательно о них сообщить.
Конечно заранее проверив что о них еще не известно например здесь #83 или где-то еще.

**Comment by completely-burned** [comment]
[CONTRIBUTING.md](../blob/master/CONTRIBUTING.md)

**Comment by user-hq** [comment]
⚠️🥃 Осторожно с крепкими напитками 😵🤢🤮🛏

**Reply by completely-burned** [reply]
Эта аллергия достала меня.

**Comment by user-hq** [comment]
У меня в сентябре аллергия из-за дождей, а из-за аллергии я болен и не адекватен.

**Comment by black-okean** [comment]
Не продвигайте проект пожалуйста, он очень сырой, не скоро будет готов, если вообще будет.
